src/meta/api/src/schema_api_impl.rs

```diff
@@ -2433,6 +2433,27 @@ impl<KV: kvapi::KVApi<Error = MetaError> + ?Sized> SchemaApi for KV {
         Ok(db_names)
     }
 
+    #[logcall::logcall("debug")]
+    #[minitrace::trace]
+    async fn get_table_name_by_id(&self, table_id: MetaId) -> Result<String, KVAppError> {
+        debug!(req :? =(&table_id); "SchemaApi: {}", func_name!());
+
+        let table_id_to_name_key = TableIdToName { table_id };
+
+        let (tb_meta_seq, table_name): (_, Option<DBIdTableName>) =
+            get_pb_value(self, &table_id_to_name_key).await?;
+
+        debug!(ident :% =(&table_id_to_name_key); "get_table_name_by_id");
+
+        if tb_meta_seq == 0 || table_name.is_none() {
+            return Err(KVAppError::AppError(AppError::UnknownTableId(
+                UnknownTableId::new(table_id, "get_table_name_by_id"),
+            )));
+        }
+
+        Ok(table_name.unwrap().table_name)
```

Comment:
shall we also check if the table has been dropped?  or use the `mget_table_names_by_ids` (to impl `get_table_name_by_id`; or directly by stream,  without introducing new method to schema api)?

Reply:
> shall we also check if the table has been dropped? or use the `mget_table_names_by_ids` (to impl `get_table_name_by_id`; or directly by stream, without introducing new method to schema api)?

Maybe no need to check, it will be checked when get_table.